pages/dashboard/Components/Sidebar.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { useUser } from '../../../lib/profile/user-data';
+import { useAuthContext } from '../../../lib/user/AuthContext';
+
+function Sidebar() {
+  const { isSignedIn } = useAuthContext();
+  const user = useUser();
+  const role = user.permissions?.length > 0 ? user.permissions[0] : '';
+  return (
```

Comment:
Minor issue with the sidebar on mobile: the text spills over the divider. The divider should be removed completely on mobile.
Also, the user type should be under the name, not to the right of it.
<img width="177" alt="Screen Shot 2021-11-06 at 11 48 00 AM" src="https://user-images.githubusercontent.com/71054105/140617323-6fb1295e-a52b-439f-bcf5-c300730672d2.png">
<img width="401" alt="Screen Shot 2021-11-06 at 11 46 30 AM" src="https://user-images.githubusercontent.com/71054105/140617301-4dc71b7f-7e6e-4fe8-aad8-2e6feccdb6e1.png">